test/org/opensolaris/opengrok/analysis/ada/AdaXrefTest.java

```diff
@@ -86,13 +86,10 @@ private int writeAdaXref(InputStream iss, PrintStream oss)
     }
 
     private String getHtmlBegin() {
-        return "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
-            "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\"\n" +
-            "    \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">\n" +
-            "<html xmlns=\"http://www.w3.org/1999/xhtml\"" +
-            " xml:lang=\"en\" lang=\"en\"\n" +
-            "      class=\"xref\">\n" +
+        return "<!DOCTYPE html>\n" +
+            "<html>\n" +
             "<head>\n" +
```

Comment:
here you discarded .xref class, are you sure that any javascript does not depend on this?

Reply:
Also why removing lang attribute?

Reply:
Hi, @tulinkry . For the test classes and data, I left the `<html>` element simple since that part of the documents is not actually under test (e.g. it is static content not truly coming from `httpheader.jspf` — where that header sets a dynamic `class`).

I removed `lang` because it did not seem correct to assert a principal natural language of `en` for OpenGrok pages since the primary content by far are `xref` pages on documents of unknown languages. Instead I thought to leave `lang` unknown to reflect this indeterminacy. Let me know if you think `en` should be restored.